src/main/java/fi/hsl/transitdata/vehicleposition/application/gtfsrt/GtfsRtOccupancyStatusHelper.java

```diff
@@ -42,102 +41,27 @@ public GtfsRtOccupancyStatusHelper(NavigableMap<Integer, GtfsRealtime.VehiclePos
-                        //If vehicle load is zero, vehicle load ratio is unavailable and the vehicle is empty
-                        return Optional.of(GtfsRealtime.VehiclePosition.OccupancyStatus.EMPTY);
-                    }
-                } catch (Exception x) {
-                    log.error("passengerCountPayload 1 failed");
-                    throw new RuntimeException("passengerCountPayload 1 failed", x);
-                }
-    
-                double vehicleLoadRatio;
-                
-                try {
-                    vehicleLoadRatio = passengerCountPayload.getVehicleCounts().getVehicleLoadRatio();
-                } catch (Exception x) {
-                    log.error("getVehicleLoadRatio() failed");
-                    throw new RuntimeException("getVehicleLoadRatio() failed", x);
-                }
-    
-                Map.Entry<Double, GtfsRealtime.VehiclePosition.OccupancyStatus> doubleOccupancyStatusEntry = null;
-                
-                try {
-                    doubleOccupancyStatusEntry = loadRatioToOccupancyStatus.lowerEntry(vehicleLoadRatio);
-                } catch (Exception x) {
-                    log.error("lowerEntry(vehicleLoadRatio) failed (vehicleLoadRatio=" + vehicleLoadRatio + ")");
-                    throw new RuntimeException("lowerEntry(vehicleLoadRatio) failed (vehicleLoadRatio=" + vehicleLoadRatio + ")", x);
-                }
-                
-                if (doubleOccupancyStatusEntry == null) {
-                    log.error("doubleOccupancyStatusEntry is null. vehicleLoadRatio=" + vehicleLoadRatio);
-                    throw new RuntimeException("doubleOccupancyStatusEntry is null. vehicleLoadRatio=" + vehicleLoadRatio);
-                }
-    
-                GtfsRealtime.VehiclePosition.OccupancyStatus occupancyStatus = null;
-                
-                try {
-                    occupancyStatus = doubleOccupancyStatusEntry.getValue();
-                } catch (Exception x) {
-                    log.error("getValue() failed");
-                    throw new RuntimeException("getValue() failed (Key=" + doubleOccupancyStatusEntry.getKey() + ", Value=" + doubleOccupancyStatusEntry.getValue() + ")", x);
-                }
-    
-                if (occupancyStatus == null) {
-                    log.error("occupancyStatus is null");
-                    throw new RuntimeException("occupancyStatus is null");
-                }
-                
-                try {
-                    return Optional.of(occupancyStatus);
-                } catch (Exception x) {
-                    log.error("passengerCountPayload 2 failed (occupancyStatus=" + occupancyStatus + ")");
-                    throw new RuntimeException("passengerCountPayload 2 failed (occupancyStatus=" + occupancyStatus + ")", x);
+                if (passengerCountPayload.getVehicleCounts().hasVehicleLoad() && passengerCountPayload.getVehicleCounts().getVehicleLoad() == 0) {
```

Comment:
What about here?
```suggestion
                if (passengerCountPayload.getVehicleCounts().hasVehicleLoad() && passengerCountPayload.getVehicleCounts().getVehicleLoad() <= 0) {
```

Reply:
More than likely this should be <= as suggested